### PR TITLE
Add redux and Toaster component

### PR DESCRIPTION
**Pull request by jgradim**
- Force indenting on switch statements
 - Allow importing scss without extension

### CODE REVIEW

File: app/actions/toasts.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+export const addToast = params => {
+  return dispatch => {
+    const id = uuid.v4();
+
+    window.setTimeout(() => dispatch(removeToast(id)), TOAST_DURATION_MS);
+
+    return dispatch(createAction(ADD_TOAST)({
+      id,
+      content: "",
+      level: ["success", "error"][Math.random() > 0.5 ? 0 : 1],
```
Comment (hugopeixoto) [inline app/actions/toasts.js]: We should probably remove this

**Review by hugopeixoto — CHANGES_REQUESTED** [review]

File: app/actions/toasts.js (hunk 1 above)
Comment (jgradim) [reply app/actions/toasts.js]: derp

**Comment by jgradim** [comment]
@hugopeixoto updated. Moved toast constants to its own file, as the `level` will need to be used throughout the app

**Comment by hugopeixoto** [comment]
I would keep them in the same file, but export them. You only use them in conjunction with `addToast`, so I see no reason to have a separate file to import.

**Comment by jgradim** [comment]
@hugopeixoto yup, good point.

**Comment by jgradim** [comment]
@hugopeixoto updated!

**Review by hugopeixoto — APPROVED** [review]

**Comment by hugopeixoto** [comment]
Go ahead with the merge

**Pull request merged**